packages/amesos2/src/Amesos2_Umfpack_FunctionMap.hpp

```diff
@@ -98,16 +98,16 @@ namespace Amesos2 {
 #ifdef HAVE_TEUCHOS_COMPLEX
 
   template <>
-  struct FunctionMap<Umfpack,std::complex<double>>
+  struct FunctionMap<Umfpack,Kokkos::complex<double>>
```

Comment:
@vbrunini thanks for the submission. I don't believe Kokkos::complex is convertible to std::complex, I think for these changes in the FunctionMap an added specialization of Kokkos::complex<T> is needed rather than replacing the std::complex<T>, could you make that change? For now, that will involve some code duplication

Reply:
@ndellingwood bump